supportedBackends/sfs/src/test/scala/cromwell/backend/impl/sfs/config/ConfigInitializationActorSpec.scala

```diff
@@ -40,5 +47,41 @@ class ConfigInitializationActorSpec extends AnyFlatSpec with CromwellTimeoutSpec
 
   }
 
-}
+  private val attributeValidationTests: TableFor4[String, String, Map[String, WomValue], ErrorOr[_]] = Table(
+    ("description", "declaration", "womValue", "expected"),
+    ("present required custom", "Int my_attr", Map("my_attr" -> WomString("1")), 1.valid),
+    ("missing required custom", "Int my_attr", Map(), "Expecting my_attr runtime attribute to be an Integer".invalidNel),
+    ("present defaulted custom", "Int my_attr = 415", Map("my_attr" -> WomString("1")), 1.valid),
+    ("missing defaulted custom", "Int my_attr = 415", Map(), 415.valid),
+    ("present optional custom", "Int? my_attr", Map("my_attr" -> WomString("1")), Option(1).valid),
+    ("missing optional custom", "Int? my_attr", Map(), None.valid),
+    ("present defaulted optional custom", "Int? my_attr = 415", Map("my_attr" -> WomString("1")), Option(1).valid),
+    ("missing defaulted optional custom", "Int? my_attr = 415", Map(), Option(415).valid),
+    ("present required memory", "Float memory_gb", Map("memory" -> WomString("3 GB")), MemorySize(3, MemoryUnit.GB).valid),
+    ("missing required memory", "Float memory_gb", Map(), "Expecting memory runtime attribute to be an Integer or String with format '8 GB'. Exception: Not supported WDL type value".invalidNel),
+    ("present defaulted memory", "Float memory_gb = 1", Map("memory" -> WomString("3 GB")), MemorySize(3, MemoryUnit.GB).valid),
+    ("missing defaulted memory", "Float memory_gb = 1", Map(), MemorySize(1, MemoryUnit.GB).valid),
+    ("present optional memory", "Float? memory_gb", Map("memory" -> WomString("3 GB")), Option(MemorySize(3, MemoryUnit.GB)).valid),
+    ("missing optional memory", "Float? memory_gb", Map(), None.valid),
```

Comment:
For the reviewers:

This `Float? memory_gb` declaration used to work but stopped at some point, along with a few other declarations listed in this test. The symptom is that on Cromwell 78 when defined as optional in the SFS backend the `memory` runtime attribute still needs to be defined for tasks or the workflow fails.

The `Float? memory_gb` declaration is both already:
- Mentioned [here in the docs](https://github.com/broadinstitute/cromwell/blob/78/docs/tutorials/HPCIntro.md#specifying-the-runtime-attributes-for-your-hpc-tasks)
- Verified on a lower level without using the wrapper `fromDeclarations` in [this other test](https://github.com/broadinstitute/cromwell/blob/78/supportedBackends/sfs/src/test/scala/cromwell/backend/impl/sfs/config/MemoryDeclarationValidationSpec.scala#L51-L52)